index.html

```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
             </div>
           </section>
             </main>
-</body>
-</html>
\ No newline at end of file
+            <footer>
+              <h2 class="contact-me">Contact me</h2>
+              <p class="footer-p">
+                If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it 
+              </p>
+              <form  class="feedback" action="https://formspree.io/html/" method="post">
```

Comment:
- [ ] As stated before kindly register in [formspree](https://formspree.io/html/) and register and use the id that you get and replace the action link with the following one and do not forget to replace the form_id also `https://formspree.io/f/{form_id}`